### ISSUE TITLE
prebuilt image for which board?

### ISSUE THREAD

**Issue by dony71** [stevenj/h3-tve#4]
You have prebuilt image "Armbian_5.41_Orangepizero_Debian_stretch_next_4.14.21.img.tar.xz"
Is this prebuilt image for Orangepizero?
Why mixer@1200000 and lcd-controller@1c0d000 is disabled?

**Comment by stevenj** [comment]
Yes, its a pre-built image for OrangePi Zero.  I used it on my pi-zero just before I posted it.  

The lcd-controller is disabled, because its not used as far as I am aware.
And from memory, there are two mixers?  And if thats right this would be the mixer for the lcd interface.  Obviously you could enable those if you wanted.
Its been a while since I played with this, and I didn't write it, I just ported forward code from one of the sunxi developers repos and kindof got it working.
But what its doing at the low level, I am not 100% sure.

**Comment by marcobrianza** [comment]
hi, what is the root password? I' trying to login via ssh

**Comment by GriDevE** [comment]
> hi, what is the root password? I' trying to login via ssh

password: 1234
